vmdb/app/models/ems_microsoft.rb

```diff
@@ -43,6 +43,9 @@ def connect(options = {})
   end
 
   def verify_credentials(_auth_type = nil, options = {})
+    silence_warnings { require 'winrm' }
```

Comment:
I don't think we need this require here.

When we finally get down to the raw_connect method, it does the require [here](https://github.com/ManageIQ/manageiq/blob/899734a8a8df29dd1edbf33e75c02ba8bbd6c97b/vmdb/app/models/ems_microsoft.rb#L24)?  Right?